tests/functional/tests/tests.yaml

```diff
@@ -0,0 +1,14 @@
+tests:
+  - openstack:
+    - tests.functional.tests.base.NoopTest
+    - tests.functional.tests.backup.BackupTest
+gate_bundles:
+  - openstack: openstack
+
+configure:
+  - openstack:
+#    - zaza.openstack.charm_tests.glance.setup.add_lts_image
```

Comment:
If any configuration is not needed, I would prefer to remove them instead of commenting them.

Reply:
removed